spec/version_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 RSpec.describe Unit do
   it 'returns the correct version' do
-    expect(Unit::VERSION).to eq '0.9.0'
+    expect(Unit::VERSION).to eq '0.9.1'
```

Comment:
This is technically kind of a breaking change for the specs since they need new ENV variables. I was thinking a minor version bump would only be appropriate if the gem itself had any breaking changes (gem consumers don't need these ENV variables)

Reply:
Version numbers are free! Since this adds functionality, Semver conventions would likely want us to use 0.10.0